### DISCUSSION TITLE
Documentation issue

### DISCUSSION THREAD

**Discussion by krishnaponkiya** [odoo/owl#1587] [General]
This documentation is not proper understable and not updated with the current version.

**Comment by sdegueldre** [comment]
If you have specific issues with the documentation feel free to open specific issues. "This whole thing sucks" isn't very actionable.

**Reply by krishnaponkiya** [reply]
As in the document everything should have in proper manner. Like you can check any other javascript framework document.
Recently, angular upgreaded the document which can understand by non tech personal also.

You may refer the links to improve the document 
https://angular.dev/

If you need any help to improve document then i can help you.
